scrapers/DarkRoomVR.yml

```
@@ -0,0 +1,32 @@
+name: "DarkRoomVR"
+sceneByURL:
+  - action: scrapeXPath
+    url:
+      - darkroomvr.com
+    scraper: sceneScraper
+xPathScrapers:
+  sceneScraper:
+    common:
+      $info: //div[@class="container"]
+    scene:
+      Title: $info//h1[@class="video-detail__title"]
+      Studio:
+        Name: 
+          fixed: DarkRoomVR
+      Date:
+        selector: $info//div[@class="video-info__text"]/div[@class="video-info__time"]/text()
+        postProcess:
+          - replace:
+              - regex: ".+ • (.*)"
+                with: $1
+          - parseDate: 2 January, 2006
+      Details:
+        selector: $info//div[@data-id="description"]/text()
+        concat: " "
```

this just gets the two descriptions and concats them
we only want the longer description
```suggestion
      Details: $info//div[@data-id="description"][span[text()="Read less"]]/text()
```